plaidml/op/op.h

```diff
@@ -337,6 +337,64 @@ inline edsl::Tensor minimum(const edsl::Tensor& X, const edsl::Tensor& Y) {
   return details::op("minimum", args).as_tensor();
 }
 
+class mvn {
+ public:
+  explicit mvn(const plaidml::edsl::Tensor& I)
+      : I_(I),
+        axes_(edsl::None()),
+        normalize_variance_(true),
+        epsilon_(1e-9),
+        across_channels_(true),
+        layout_("NHWC") {}
+
+  mvn& axes(const std::vector<int64_t>& axes) {
+    // negative axes interpreted as in numpy
+    axes_ = edsl::make_tuple(axes);
+    return *this;
+  }
+
+  mvn& remove_axes_parameter() {
+    // Restores axes to the default of being unused (meaning use `across_channels` & `layout` instead)
+    axes_ = edsl::None();
+    return *this;
+  }
```

Comment:
Hmm, do we need this? What's the thinking here?

Reply:
Eh, probably not really necessary. The thinking was that without this, once `axes` was set there was no way to un-set it except creating a new instance of the `mvn` class. That said, I don't really see why anyone would need to set and then un-set the `axes` parameter.